electrode-ota-server-dao-mariadb/test/ElectrodeOtaDaoRdms-spec.ts

```diff
@@ -1618,14 +1618,14 @@ describe("Data Access via RDBMS", function() {
             "v2"
           );
         });
-        it("will return no release for matching tag but not appVersion", () => {
+        it.skip("will return no release for matching tag but not appVersion", () => {
```

Comment:
fix

Reply:
updated the test as per signature of the getNewestApplicablePackage method.